sway/tree/view.c

```diff
@@ -639,12 +640,18 @@ static void view_subsurface_create(struct sway_view *view,
 	view_child_init(child, NULL, view, subsurface->surface);
 }
 
+static void view_child_damage(struct sway_view_child *child, bool whole) {
+	int sx, sy;
+	child->impl->get_root_coords(child, &sx, &sy);
```

Comment:
What about subsurfaces?

Reply:
Yeah, it segfaults. I should have checked that :\

Reply:
Should be good now.